release.sh

```diff
@@ -53,21 +54,7 @@ echo
 if [[ $REPLY =~ ^[Yy]$ ]]
 then
 	sed -i -e "s/\"version\":.*$/\"version\": \"$VERSION_NO_V\",/" bower.json
-fi
-
-read -p "Bump build number? " -n 1 -r
-echo
-if [[ $REPLY =~ ^[Yy]$ ]]
-then
 	sed -i -e "s/\"build\":.*$/\"build\": \"$VERSION_NO_V\"/" bower.json
-	sed -i -e "s/\"build\":.*$/\"build\": \"$VERSION_NO_V\"/" package.json
```

Comment:
I merged it into line 42 so that all package.json changes are together, same for bower.json